docs/install-cicero.md

```diff
@@ -0,0 +1,40 @@
+# Install Cicero
+
+## How to install Cicero locally
+
+### Clone repository locally
+
+```
+git clone https://github.com/input-output-hk/cicero.git
+```
+
+### Provide github token with read:org, repo flags
+
+```
+cat >> ~/.netrc << EOF
+machine github.com
+login github_username
+password github_token
+
+machine api.github.com
+login github_username
+password github_token
+EOF
+```
+
```

Comment:
Could it be that the `nix develop` step is missing here?

Reply:
That's right. will add it to the next version.